lib/iev/db.rb

```diff
@@ -55,7 +55,7 @@ def open_cache_biblio(dir, global: true)
       db = DbCache.new dir
       if global
         unless db.check_version?
-          FileUtils.rm_rf dir + "/."
+          FileUtils.rm_rf(Dir.glob(dir + '/*'), secure: true)
```

Comment:
We probably should not hard code `/*` but just `*`, and use `File.join`?

Reply:
Fixed